services/app-api/utils/validation/completionStatus.test.ts

```diff
@@ -175,6 +175,32 @@ describe("Completion Status Tests", () => {
       expect(result).toMatchObject({});
     });
 
+    test("If user has not added an ILOS, they're not required to complete that section", async () => {
+      const testData = {};
+      const formTemplate = {
+        routes: [
+          {
+            name: "D: Plan-Level Indicators",
+            path: "/mcpar/plan-level-indicators",
+            children: [
+              {
+                name: "ILOS",
+                path: "/mcpar/plan-level-indicators/ilos",
+                entityType: "plans",
+                pageType: "drawer",
+                drawerForm: {
+                  id: "dpa",
+                  fields: [],
+                },
+              },
+            ],
+          },
+        ],
+      };
+      const result = await calculateCompletionStatus(testData, formTemplate);
+      expect(result).toMatchObject({});
```

Comment:
What does an empty `{}` mean? Just curious.

Reply:
I based that off of the previous unit tests, but I asked @ntsummers1 and I _belieeeeve_ it's because of this [line](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/7a11625b38c7f1d46d9be37a25732d4104a87ae7/services/app-api/utils/validation/completionStatus.ts#L230) where the `completionDict` will be coming back empty because the field data itself is empty.